test/integration/broken/pool-simulator-tests.js

```diff
@@ -784,7 +770,8 @@ describe("pool", function () {
                 await promiseUtils.delay(20);
 
                 // Shutdown while there are queries in progress
-                await client.shutdown();
+                // Note: this will likely no longer work in our driver
+                // await client.shut down();
```

Comment:
:wrench:

Reply:
1. `shut down` -> `shutdown`
2. Why `likely`? It's just not supported, let's be blunt clear about that.